qiskit_nature/operators/second_quantization/second_quantized_op.py

```diff
@@ -10,84 +10,47 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""The Second-Quantized Operator."""
+"""The Sum Operator base interface."""
```

Comment:
Why is this changed? Should this be the Second-Quantizer Operator interface?

Reply:
This is a limitation of how git diffes these files. Here, the old `SecondQuantizedOp` is being compared against `ParticleOp` (which is now called `SecondQuantizedOp`).
What actually happened is:
- old `operators/second_quantization/second_q_op.py` got removed
- `operators/second_quantization/particle_op.py` moved to `operators/second_quantization/second_q_op.py`
But git diff shows it as:
- `operators/second_quantization/particle_op.py` got removed
- and diffes the changes on the `second_q_op.py` file

In short: I didn't change it, but git shows it as such because of how the files were renamed.